components/consent/consent.css

```diff
@@ -0,0 +1,71 @@
+@keyframes slideUp {
+  0% {
+    transform: translateY(100%);
+    opacity: 0.5;
+  }
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+amp-consent {
+  background: var(--color-white);
+  box-shadow: rgba(0, 0, 0, 0.26) 0 0 19px 0;
+  animation: .5s ease-out 0s 1 slideUp;
+}
+
+amp-consent .ampstart-btn {
+  width: 9.5em;
+}
+
+.ampstart-consent-popup p a {
+  color: #b60845;
+}
+
+/* phones */
+@media (--breakpoint-xs) {
+  amp-consent a.ampstart-consent-close-btn {
+    font-size: 22px;
+    top: 15px;
+    right: 13px;
+  }
+  
+  .ampstart-consent-popup div {
+    flex-flow: column wrap;
+    align-items: center;
+    margin: 12px 25px 15px 12px;
+  }
+
+  .ampstart-consent-popup p {
+    padding: 5px 20px 5px 10px;
+    margin: 0 0 10px 0;
+  }
+}
+
+/* tablets and desktop */
+@media (--breakpoint-sm) {
+ .ampstart-consent-popup a.ampstart-consent-close-btn {
+    font-size: 25px;
+    left: 30px;
+  }
+
+  .ampstart-consent-popup div {
+    flex-flow: row;
+    height: 100px;
+    margin: 0 40px;
```

Comment:
use spacing vars for margins (unless they are one off)

Reply:
Ok!  I've added vars for spacing, line height, and font size where it seemed to me very close to the existing design and where I could apply them consistently.  Let me know if you want moooooreeeeeee...

Reply:
in general all margins and paddings should use the variables (unless they  are exemptions by design)